web-admin/src/routes/[organization]/[project]/canvas/[dashboard]/+page.svelte

```diff
@@ -100,6 +116,6 @@
 
 {#if isCanvasReconcilingForFirstTime(canvasResource)}
   <DashboardBuilding />
-{:else}
-  <CanvasDashboardEmbed resource={canvasResource} />
+{:else if !$categorizedBookmarksStore.isPending}
```

Comment:
This negation is a little confusing. I'd expect to see `if (isPending) { <loadingState />} else { <Component />}`